File: backend-model/src/main/java/outcobra/server/model/dto/TimetableEntryDto.java
package outcobra.server.model.dto;

import outcobra.server.model.TimetableEntry;
import outcobra.server.model.interfaces.MappableDto;
import outcobra.server.model.interfaces.Mapper;

public class TimetableEntryDto implements MappableDto<TimetableEntryDto, TimetableEntry> {
    @Override
    public Mapper<TimetableEntryDto, TimetableEntry> getMapper() {
        throw new UnsupportedOperationException("not implemented");
    }

    @Override
    public TimetableEntry toEntity() {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: backend/src/main/kotlin/outcobra/server/model/dto/emptyDto/TimetableEntryDto.kt
package outcobra.server.model.dto.emptyDto

import outcobra.server.model.TimetableEntry
import outcobra.server.model.mapper.MappableDto
import outcobra.server.model.mapper.Mapper

class TimetableEntryDto : MappableDto<TimetableEntryDto, TimetableEntry> {
    override fun getMapper(): Mapper<TimetableEntryDto, TimetableEntry> {
        throw UnsupportedOperationException("not implemented")
    }

    override fun toEntity(): TimetableEntry {
        throw UnsupportedOperationException("not implemented")
    }
//TODO Implement
}